package/kedro_viz/server.py

```diff
@@ -107,16 +107,19 @@ def run_viz(port=None, line=None) -> None:
     display(HTML(wrapper))
 
 
-def get_data_from_kedro():
-    """ Get pipeline data from Kedro and format it appropriately """
+def get_data_from_kedro(pipeline):
```

Comment:
could you please add catalog as well for future proofing

Reply:
Does the name `get_data_from_kedro` still make sense since the objects are now passed in?  I made the catalog argument optional for now  since it is not being utilized.  Let me know your thoughts.

Reply:
if it's optional now then code that uses it in the future will have to be conditional on it being provided or it will still be a compatibility break

Reply:
the name doesn't really make sense, format_pipeline_data might be better

Reply:
Those changes are now in